smalltalksrc/VMMakerTests/VMImageHeaderWritingTest.class.st

```diff
@@ -167,6 +157,16 @@ VMImageHeaderWritingTest >> testWritingImageWritesCorrectImageFormat [
 	self assert: header imageFormat equals: interpreter imageFormatVersion
 ]
 
+{ #category : #tests }
+VMImageHeaderWritingTest >> testWritingImageWritesCorrectImageVersion [
+
+	| header |
+
+	header := self readHeader.
+
+	self assert: header imageVersion equals: interpreter getImageVersion
+]
+
```

Comment:
??

Reply:
I added that test, in the setup sets a value to check (in the interpreter) and that should be preserved in the header